### PR TITLE
Unset line height on reacji.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/hotpotatosquad, there's probably a better way to do this. I'm open to suggestions. It removes the line height from reacji to fully center them in the container. It's most obvious on square emoji (`all-the-things` in this example).

Before:
Screen Shot 2020-04-08 at 11 47 58

After:
Screen Shot 2020-04-08 at 11 47 42

### CODE REVIEW

**Comment by adamjspooner** [comment]
Inline example:
Screen Shot 2020-04-08 at 13 48 03

**Comment by adamjspooner** [comment]
Mobile (after `0275a10`):
Screen Shot 2020-04-08 at 15 34 51

(It doesn't seem to have broken anything on mobile, but @hyperobject might want to take a look.)

File: shared/common-adapters/custom-emoji.native.tsx [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
   const {size, src} = props
   return (
-    <FastImage
-      source={{uri: src}}
-      style={{
-        height: size,
-        width: size,
-      }}
-    />
+    <Box2 direction="vertical" style={styles.container}>
```
Comment (hyperobject) [inline shared/common-adapters/custom-emoji.native.tsx:10]: This `Box2` needs an explicit height set to not break android, should be able to reuse the same width and height style from the image!

Comment (adamjspooner) [reply shared/common-adapters/custom-emoji.native.tsx]: Updated! It still looks good on iOS. I'm still working on testing on Android.

File: shared/common-adapters/custom-emoji.native.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import {Box2} from './box'
 const CustomEmoji = (props: Props) => {
   const {size, src} = props
   return (
-    <Box2 direction="vertical" style={styles.container}>
+    <Box2 direction="vertical" style={Styles.collapseStyles([{height: size}, styles.container])}>
```
Comment (hyperobject) [inline shared/common-adapters/custom-emoji.native.tsx:10]: Oh whoops I think I told you the wrong thing, looks like you need `height` and `width` set on the box to stop android crashes.

**Pull request merged**